components/BackButton/BackButton.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+const ButtonStyle = styled.button`
+  border: 1px solid black;
+  padding: 1rem;
+  margin: 0.5rem;
+  border-radius: 50%;
+  background: grey;
+  text-decoration: none;
+  color: white;
+`;
+
+export default function BackButton({ btnContent, handleClick }) {
```

Comment:
Please do not use abbreviations as they make the code harder to read. 
```suggestion
export default function BackButton({ buttonContent, handleClick }) {
```